### PR TITLE
Implement get_image function for softlayer

### PR DESCRIPTION
**Pull request by Fran-Rg**
### Description

`libcloud/libcloud/compute/base.py`propose a function `get_image` which isn't implement in softlayer.

### Status

- done, ready for review

### Checklist (tick everything that applies)

- [ ] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [ ] Documentation
- [ ] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

File: libcloud/compute/drivers/softlayer.py [hunk 1]
```diff
@@ -432,6 +433,24 @@ def list_images(self, location=None):
...
+        :type image_id: ``str``
+
+        :return: A NodeImage object
+        :rtype: :class:`NodeImage`
+
+        """
+        images = self.list_images()
+        images = [image for image in images if image.id == image_id]
+        if len(images) < 1:
+            raise LibcloudError('could not find the image with id %s' % image_id)
```
Comment (tonybaloney) [inline libcloud/compute/drivers/softlayer.py]: this line is slightly too long and failing the linter

**Review by tonybaloney — CHANGES_REQUESTED** [review]

**Comment by tonybaloney** [comment]
thanks for your contribution @IVBakker can you please add a complimentary test

**Comment by Fran-Rg** [comment]
I'll make the linter happier and look for a test coverage.

**Comment by Fran-Rg** [comment]
@tonybaloney The CI build is failing on:
```
 File "../contrib/generate_provider_feature_matrix_table.py", line 312, in generate_providers_table
    base_method = base_methods[method_name]
KeyError: 'create_balancer'
ERROR: InvocationError: '/home/travis/build/apache/libcloud/.tox/docs-travis/bin/python ../contrib/generate_provider_feature_matrix_table.py'
```
I don't think this is related to my change.

**Comment by tonybaloney** [comment]
:+1: thanks @IVBakker no the CI failure is unrelated.

**Pull request closed without merging**